eng/testing/linker/SupportFiles/Directory.Build.targets

```diff
@@ -1,18 +1,20 @@
 <Project>
   <Target Name="RestoreProject">
     <MSBuild Projects="$(MSBuildProjectFullPath)"
-             Properties="Configuration=$(Configuration)"
+             Properties="Configuration=$(Configuration);ForceEvaluation=true"
```

Comment:
Do you mind sharing why we need to set this property?

Reply:
This is to force the project to be re-evaluated in the restore step. Since we're calling it from within the same MSBuild node, if we don't do this it won't re-evaluate the project and you would get an error that the Framework package was not found if it is not in the nuget cache or in the SDK fallback folder.